<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1221</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="IP_edit">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>40</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>20</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter IP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>70</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Command</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_meas_but">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>180</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Meas.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="close_but">
    <property name="geometry">
     <rect>
      <x>1072</x>
      <y>180</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>One Meas.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connect_but">
    <property name="geometry">
     <rect>
      <x>1072</x>
      <y>140</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disconnect_but">
    <property name="geometry">
     <rect>
      <x>1072</x>
      <y>100</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_but">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>560</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="one_frame_but">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>140</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>One Meas.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Graph</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Port_edit">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QPushButton" name="contin_meas_but">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>100</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>âˆž</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="graph_lb" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>891</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="OneMeas_lb">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
